content/docs/guides/webpack/contents.lr

```diff
@@ -34,12 +34,29 @@ Now you need to configure webpack.  The plugin expects a webpack project in the
 
 ### `package.json`
 
-This file instructs `npm` which packages we will need.  All we need for a
-start is to create an almost empty file:
+This file instructs `npm` which packages we will need.
+
+[npm-init :ext](https://docs.npmjs.com/cli/init) is a command-line tool to
+interactively create a ``package.json`` file.
+
+```
+$ npm init
+```
+
+This will ask you a bunch of questions (invoke with ``--yes`` to use default
+values) and then generate the file for you:
```

Comment:
Can you indicate that the provided JSON file is just an example, and shouldn't be copy-pasted verbatim?